hack/test-assets.sh

```diff
@@ -26,7 +26,7 @@ pushd "${OS_ROOT}" > /dev/null
   echo "Validating checked in bindata.go is up to date..."
   if ! assetdiff=$(diff -u _output/test/assets/bindata.go pkg/assets/bindata.go) ; then
 
-    echo "$assetdiff" | head -c 1000
+    echo "$assetdiff" | head -n 10
```

Comment:
Binary assets like images tend to be the problematic ones, and get put on a single line, so the first 10 lines gets me both lines from the diff where binary content of one file differs